src/login/utilities.ts

```diff
@@ -0,0 +1,4 @@
+export function isValidateRedirect(url: string): boolean {
```

Comment:
I think this just needs to be called `isValidRedirect` but good otherwise!

Reply:
Oops. I was going to call this `validateRedirect` at first and have it raise an Exception at first but decided to return a boolean instead.